is3106backend/users/views.py

```diff
@@ -123,13 +114,17 @@ def protected_user_update_view(request, pk):
             user = CustomUser.objects.get(pk=pk)
             if name: user.name = name
             if email: user.email = email
-            if hasattr(VendorUser.objects.get(user=user), 'is_vendor'):
-                vendor = VendorUser.objects.get(user=user)
-                if vendor_name: vendor.vendor_name = vendor_name
-                vendor.save()
+            try:
+                if hasattr(VendorUser.objects.get(user=user), 'is_vendor'):
```

Comment:
no need for this, this always evaluates to true. just get VendorUser will do

Reply:
okay will do